CHANGELOG.md

```diff
@@ -16,7 +17,7 @@
 ## 3.3.0 (2021-05-26)
 
 - Allow setting a custom timestamp on the breadcrumbs (#1193)
-- Add option `ignore_tags` to `IgnoreErrorsIntegration` in order to ignore exceptions by tags values. (#1201)
+- Add option `ignore_tags` to `IgnoreErrorsIntegration` in order to ignore exceptions by tags values (#1201)
```

Comment:
outside scope of this merge request, move to a separate pull request

Reply:
I'm just removing the final dot to keep consistency with the rest of the points of the list, it's not worth opening a separate PR for this 😅

Reply:
it's not about worthiness, it's about quality. you don't stuff unrelated changes together.

please read about atomic commits:
- https://www.freshconsulting.com/insights/blog/atomic-commits/

so, now consider your changeset needs to be reverted, but since it's all same commit, your unrelated change will be reverted too with `git revert`.

also, unrelated noise makes PR more readable and the style change PR gets merged very likely sooner as it's not affecting functionality.

Reply:
I understand your motivation and I agree with your reasoning when talking about big changes not related to the PR's purpose, but commenting that adding a dot to a phrase is something that should be done separately has no value for the code review in my opinion and it's just nitpicking